code-challenges.js

```diff
@@ -9,67 +9,207 @@
 // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
+//Pseudocode
+//create a function
+//argument is an array
+//use .map to mutiply every number in array by 5
+//no for loop needed due to functionality of .map
+//.map cycles through every element in the array
+//console.log(function(array)); to test
+
 var myMapNumbers1 = [87, 6, 28, 16, 2]
 var myMapNumbers2 = [8, -7, 0, 6, 12]
 
-
-
+const mapMultFive = (array) => {
+  return  array.map(value => {
+    return value * 5
+  })
+}
+console.log(mapMultFive(myMapNumbers1));
+console.log(mapMultFive(myMapNumbers2));
 
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
+//Pseudocode
+//create a function
+//argument will be a string in a variable
+//break up letters into an array using .split
+//using newly split array, use .filter to sort out vowels
+//.filter will return an array, so use .join to turn array into a string
+//return new string
+//console.log(function(string)); to test
+
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
+const vowelRem = (string) => {
+  let stringLetters = string.split("")
+  return stringLetters.filter(letter => {
+    return letter !== "a" && letter !== "A" && letter !== "e" && letter !== "E" && letter !== "i" && letter !== "I" && letter !== "o" && letter !== "O" && letter !== "u" && letter !== "U"
+  }).join("")
+}
+console.log(vowelRem(stringWithVowels1));
+console.log(vowelRem(stringWithVowels2));
 
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string" "mCdngMstr"
 
+//Pseudocode
+//create a function
+//argument will be a variable
+//use if else to check if input is string or otherwise
+//if something other than string return string saying input is not a string
+//if string is input then complete following procedure to remove vowels
+//break up letters into an array using .split
+//using newly split array, use .filter to sort out vowels
+//.filter will return an array, so use .join to turn array into a string
+//return new string
+//console.log(function(variable)); to test
+
+
 var refactorTester1 = true
 var refactorTester2 = 42
 var refactorTester3 = "IAmACodingMaster"
 
-
+const vowelRem2 = (input) => {
+  if(typeof input === "string"){
+    let stringLetters = input.split("")
+    return stringLetters.filter(letter => {
+      return letter !== "a" && letter !== "A" && letter !== "e" && letter !== "E" && letter !== "i" && letter !== "I" && letter !== "o" && letter !== "O" && letter !== "u" && letter !== "U"
```

Comment:
For cleaner code, consider using an array to hold letters that need to be removed